public/static/js/waterfall.js

```diff
@@ -61,7 +61,7 @@ class JiraLink extends React.Component {
         let hasSpace = i !== (tokens.length - 1);
         let maybeSpace = hasSpace ? ' ': '';
         if(token.match(JIRA_REGEX)) {
-          let jiraLink = jiraHost+"/browse/"+token;
+          let jiraLink = "https://"+jiraHost+"/browse/"+token;
```

Comment:
The fix has already been made in jsx/waterfall.jsx. This diff is generated by rerunning grunt, and should be merged.